scripts/notifications.coffee

```diff
@@ -40,7 +39,17 @@ GRAFANA = {
   tags: {}
 }
 
-PRESETS = ['hackers', 'mediums', 'help']
+PRESETS = ['buildgt', 'mediums', 'help', 'approve', 'abort']
```

Comment:
Looks like a bit of a merging issue, we want the `hackers` preset (buildgt is old).